modules/govuk_jenkins/templates/jobs/deploy_app_downstream.yaml.erb

```diff
@@ -0,0 +1,53 @@
+---
+- job:
+    name: Deploy_App_Downstream
+    display-name: Deploy_App_Downstream
+    project-type: freestyle
+    description: "Kicks off an appliction deploy in a downstream environment"
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          artifact-num-to-keep: 5
+    builders:
+      - shell: |
+        if ! [ "$TAG" = "release_*" ]; then
+          echo "Skipping downstream deploy for non-release tag $TAG"
+          exit 0
+        fi
+
+        <% if @deploy_url %>
+        JSON="{\"parameter\": [{\"name\": \"TARGET_APPLICATION\", \"value\": \"$TARGET_APPLICATION\"}, {\"name\": \"TAG\", \"value\": \"$TAG\"}, {\"name\": \"DEPLOY_TASK\", \"value\": \"$DEPLOY_TASK\"}, {\"name\": \"NOTIFY_RELEASE_APP\", \"value\": \"true\"}, {\"name\": \"SLACK_NOTIFICATIONS\", \"value\": \"true\"}], \"\": \"\"}"
+
+        # Deploy to AWS Integration environment
```

Comment:
This comment isn't correct any more, as it could be going to staging, right?

Reply:
Oops - fixed 👍. 

(The original author was really keen on the word 'Integration'!)